handler/core/src/main/java/io/knotx/fragments/handler/action/cb/CircuitBreakerAction.java

```diff
@@ -34,6 +34,8 @@
 import io.vertx.core.Handler;
 import io.vertx.core.Promise;
 import io.vertx.core.json.JsonObject;
+import java.util.HashSet;
+import java.util.Set;
 import java.util.concurrent.atomic.AtomicInteger;
 
 class CircuitBreakerAction implements Action {
```

Comment:
Please provide unit tests for factory:
- no errorTransitions are defined in options - _error transition should be used
- custom error transition is defined